src/MongoDB/ReadPreference.c

```diff
@@ -733,6 +755,12 @@ void php_phongo_readpreference_init_ce(INIT_FUNC_ARGS) /* {{{ */
 	zend_declare_class_constant_long(php_phongo_readpreference_ce, ZEND_STRL("RP_NEAREST"), MONGOC_READ_NEAREST TSRMLS_CC);
 	zend_declare_class_constant_long(php_phongo_readpreference_ce, ZEND_STRL("NO_MAX_STALENESS"), MONGOC_NO_MAX_STALENESS TSRMLS_CC);
 	zend_declare_class_constant_long(php_phongo_readpreference_ce, ZEND_STRL("SMALLEST_MAX_STALENESS_SECONDS"), MONGOC_SMALLEST_MAX_STALENESS_SECONDS TSRMLS_CC);
+
+	zend_declare_class_constant_string(php_phongo_readpreference_ce, ZEND_STRL("PRIMARY"), PHONGO_READ_PRIMARY TSRMLS_CC);
```

Comment:
As in #1061, I'm exposing constants here. Since the class name is `ReadPreference`, I decided to avoid using any prefix. This also resolves a naming conflict with the `RP_` constants that expose the integer values for read preference modes.

I'm not too happy about having two very similar constants (`RP_PRIMARY` and `PRIMARY`) in the same class, as this may confuse users. We don't need to expose these values as constants now and may opt to only do that in 2.0 when the `RP_` constants are dropped.

Reply:
I'm OK with adding the constants now, as it's consistent with what we're already doing in [ReadConcern](https://www.php.net/manual/en/class.mongodb-driver-readconcern.php). If we don't already have tickets to deprecate and remove the `RP_` constants, feel free to create those.

Reply:
Deprecation is tracked in [PHPC-1489](https://jira.mongodb.org/browse/PHPC-1489). I've added tests for all constants, including int ones for good measure.